x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/exporter/local/LocalExporterResourceIntegTests.java

```diff
@@ -81,11 +82,14 @@ private static BytesReference generateTemplateSource(final String name, final In
                 .field("index.number_of_replicas", 0)
             .endObject()
             .startObject("mappings")
-                .startObject("doc")
+                // Still need use type, RestPutIndexTemplateAction#prepareRequestSource has logic that adds type if missing
+                .startObject(MapperService.SINGLE_MAPPING_NAME)
```

Comment:
@jtibshirani In your offline discussion you mentioned:

> Internal elasticsearch products like monitoring and security have been using 'doc' as a type name, and in particular create index templates using the custom type 'doc'.

Given that, should the argument to `startObject` here be `MapperService.SINGLE_MAPPING_NAME`, which would evaluate to `_doc` _or_ should it be `"doc"`?

Reply:
I think for now we should keep it the same as before (using `doc`), until we come to a clear conclusion in that discussion around updating the monitoring templates.